packages/formstr-app/src/containers/MyForms/components/DeleteForm/index.tsx

```diff
@@ -0,0 +1,38 @@
+import {IDeleteFormsLocal, IDeleteFormsTrigger} from "./types";
+import {DeleteFilled} from "@ant-design/icons";
+import {Button, Modal} from "antd";
+import {useState} from "react";
+import {getItem, LOCAL_STORAGE_KEYS, setItem} from "../../../../utils/localStorage";
+import {ILocalForm} from "../Local/typeDefs";
+
+function DeleteConfirmationLocal({formId, onCancel, onDeleted}: IDeleteFormsLocal) {
+    const onDeleteForm = () => {
+        let localForms = getItem<ILocalForm[]>(LOCAL_STORAGE_KEYS.LOCAL_FORMS) ?? [];
+        localForms = localForms.filter(v => v.key !== formId)
+        setItem(LOCAL_STORAGE_KEYS.LOCAL_FORMS, localForms)
+        onDeleted()
+    }
+    return <Modal title={'Are you sure you want to delete the form?'} open onOk={onDeleteForm} onCancel={onCancel}>
```

Comment:
are you sure you want to remove this form from device storage?

Reply:
Yup
And we should as well. The user thinks its deleted from the device and we should honour that.

And we should also give the export option here anyways(doesn't matter if user is deleting from nostr or local)

Reply:
i meant, the message should read "are you sure you want to remove this form from device storage?"